src/status_im2/common/standard_auth/keycard/view.cljs

```diff
@@ -0,0 +1,178 @@
+         :blur?       false
+         :delete-key? false
+         :left-action :none}]])))
+
+(defn this-is-not-keycard-sheet
+  []
+  [rn/view {:style style/container}
+   [rn/view {:style style/inner-container}
+    [quo/button
+     {:container-style style/close-button
+      :type            :grey
+      :icon-only?      true
+      :size            32
+      :on-press        #(rf/dispatch [:hide-bottom-sheet])} :i/close]
+
+    [quo/text
+     {:accessibility-label :this-is-not-keycard-text
+      :size                :heading-1
+      :weight              :semi-bold}
+     (i18n/label :t/this-is-not-keycard)]
+
+    [quo/text
+     {:accessibility-label :make-sure-text
+      :size                :paragraph-1
+      :weight              :medium
+      :style               style/secondary-text} (i18n/label :t/make-sure-scanned-keycard)]]
+
+   [rn/image
+    {:resize-mode :contain
+     :style       {:aspect-ratio  1
+                   :height        503
+                   :margin-bottom 20
+                   :align-self    :center}
+     :source      (resources/get-image :this-is-not-keycard)}]
+
+   [rn/view {:style style/try-again-button}
+    [quo/button
+     {:type :primary}
+     (i18n/label :t/try-again)]]])
+
+(defn keycard-locked-sheet
+  []
+  [rn/view
+   {:style  style/container
+    :height (:height (rn/get-window))}
+   [rn/view {:style style/inner-container}
+    [quo/button
+     {:container-style style/close-button
+      :type            :grey
+      :icon-only?      true
+      :size            32
+      :on-press        #(rf/dispatch [:hide-bottom-sheet])} :i/close]
+
+    [quo/text
+     {:accessibility-label :keycard-locked-text
+      :size                :heading-1
+      :weight              :semi-bold}
+     (i18n/label :t/keycard-locked)]
+
+    [quo/text
+     {:accessibility-label :unlock-keycard-text
+      :size                :paragraph-1
+      :weight              :medium
+      :style               style/secondary-text} (i18n/label :t/unlock-keycard-to-use)]
+
+    [rn/view {:style style/keycard}
+     [keycard/keycard
+      {:holder-name "Alisher"
+       :locked?     true}]]
+
+    [quo/button
+     {:type :primary}
+     (i18n/label :t/unlock-keycard)]
+
+    [quo/divider-label
+     {:tight?          false
+      :container-style style/divider} (i18n/label :t/other-options)]
+
+    [quo/button
+     {:type            :ghost
+      :size            20
+      :icon-left       :i/refresh
+      :container-style style/reset-keycard-button} (i18n/label :t/factory-reset-keycard)]]])
+
+(defn wrong-keycard-sheet
+  []
+  [rn/view {:style style/container}
+   [rn/view {:style style/inner-container}
+    [quo/button
+     {:container-style style/close-button
+      :type            :grey
+      :icon-only?      true
+      :size            32
+      :on-press        #(rf/dispatch [:hide-bottom-sheet])} :i/close]
+
```

Comment:
Although readability is very desirable, in `status-mobile` we don't usually add blank lines to separate components. There are exceptions, for example when the components are surrounded by logic checks with `cond*`, but the functions in this namespace are simple enough.

Reply:
@ilmotta Got it! Removed the extra lines.